src/program/lwaftr/setup.lua

```diff
@@ -58,6 +63,9 @@ function lwaftr_app(c, conf)
       error("One or both of the 'next_hop' values must be specified")
    end
 
+   -- Validate configuration file we've been given.
+   validate_config(conf)
+
```

Comment:
I think we probably don't want to do this here.  I mean, this function will be called on any configuration change, including incremental updates, right?  We probably don't want to cause an O(n) verification when our O(1) update already verified the thing.  So when do we need to verify?  Perhaps only when loading the initial configuration from disk.